src/osd/scheduler/OpSchedulerItem.h

```diff
@@ -156,6 +169,16 @@ class OpSchedulerItem {
   utime_t get_start_time() const { return start_time; }
   uint64_t get_owner() const { return owner; }
   epoch_t get_map_epoch() const { return map_epoch; }
+  dmc::ReqParams get_qos_req_params() const { return qos_req_params; }
+  void set_qos_req_params(dmc::ReqParams qparams) {
+    qos_req_params = qparams;
+  }
+  const client_qos_params_t& get_qos_profile_params() const {
+    return qos_profile_params;
+  }
+  void set_profile_params(client_qos_params_t qparams) {
```

Comment:
This also doesn't appear to have a user.

Reply:
Removed for now as there is no user.